events/giveaways/giveawayEnded.js

```diff
@@ -1,4 +1,5 @@
 const { MessageEmbed, MessageButton, MessageActionRow } = require("discord.js")
+const Discord = require('discord.js');
 
 module.exports = {
   async execute(giveaway, winners) {
```

Comment:
Same thing here about renaming `giveaway` to whatever you want. `winners` too. Make it more idiomatic